app/views/content/train-to-be-a-teacher/promos/_adviser-promo-apply.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
   <% promo.with_left_section(classes: %w[tta-background]) %>
   <% promo.with_right_section(heading: "Get free one-to-one support") do %>
-      <p>Maximise your chances of submitting a successful application with the support of a dedicated adviser with years of teaching experience. Chat to an adviser through phone, text or email.</p>
+      <p>Maximise your chances of submitting a successful application with the support of a dedicated adviser with years of teaching experience. Chat by phone, text or email, as little or as often as you need.</p>
```

Comment:
Presume we are keeping this for the time being as it is used on `/train-to-be-a-teacher/school-placements`?

But that it will be deleted when that section is addressed?

Reply:
Yes, exactly! I'll remove it with the train to teach updates.